# HG changeset patch
# User Rik <rik@octave.org>
# Date 1372909085 25200
#      Wed Jul 03 20:38:05 2013 -0700
# Node ID 068f26c93ac7bf395b7c51214cd3b2d78ce2732e
# Parent  8fbc09eded8c4e49e716adba3400a488c098fd02
octave-link.cc: Fix compilation warning about signed/unsigned comparison.

* libinterp/corefcn/octave-link.cc(__octave_link_file_dialog__): use
unsigned int in for loop because size() from std::list returns size_t
which is unsigned.

diff --git a/libinterp/corefcn/octave-link.cc b/libinterp/corefcn/octave-link.cc
--- a/libinterp/corefcn/octave-link.cc
+++ b/libinterp/corefcn/octave-link.cc
@@ -250,17 +250,17 @@ Undocumented internal function.\n\
           else
             {
               // Multiple files.
               nel = items_lst.size ();
               Cell items (dim_vector (1, nel));
 
               std::list<std::string>::iterator it = items_lst.begin ();
 
-              for (int idx = 0; idx < items_lst.size ()-2; idx++)
+              for (unsigned int idx = 0; idx < items_lst.size ()-2; idx++)
                 {
                   items.xelem (idx) = *it;
                   it++;
                 }
 
               retval(0) = items;
               retval(1) = *it++;
               retval(2) = atoi (it->c_str ());
